torch/csrc/jit/pybind_utils.cpp

```diff
@@ -0,0 +1,23 @@
+#include <torch/csrc/utils/pybind.h>
```

Comment:
You're missing

```
#include <torch/csrc/jit/pybind_utils.h>
```

That'll fix the runtime dlopen linking error

Reply:
Awesome :O